ui/imports/shared/popups/ConfirmationDialog.qml

```diff
@@ -9,7 +9,6 @@ import StatusQ.Popups 0.1
 
 StatusModal {
     id: confirmationDialog
-    anchors.centerIn: parent
```

Comment:
`StatusModal` has a good default to center over the app window (doesn't have to be the parent necessarily)